learning_journal/models/mymodel.py

```diff
@@ -0,0 +1,26 @@
+from sqlalchemy import (
+    Column,
+    Index,
+    Integer,
+    Text,
+    UnicodeText,
+    Date
+)
+
+from .meta import Base
+import datetime
+
+
+def _now():
+    return datetime.datetime.now()
+
+
+class Journal(Base):
```

Comment:
I would argue that this should be called `Entry`, rather than `Journal`, but I wouldn't argue it much.